.github/workflows/build-push-template.yml

```
@@ -0,0 +1,48 @@
+name: Build and Push template
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build-push-template:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build and push the recipes Docker image
```

```suggestion
    - name: Build and push Docker image for Substrate Playground
```

Because this is the recipes repo, we don't need to mention the recipes. We _should_ mention the playground though because it is the external thing we're integrating with.